src/OpenLoco/Windows/TerraForm.cpp

```diff
@@ -31,7 +31,7 @@ namespace OpenLoco::Ui::Windows::Terraform
     static loco_global<std::uint8_t[10], 0x00500775> _byte_500775;
     static loco_global<int16_t, 0x0052337A> _dragLastY;
     static loco_global<Ui::WindowType, 0x00523392> _toolWindowType;
-    static loco_global<int8_t, 0x00523393> _currentTool;
+    static loco_global<CursorId, 0x00523393> _currentToolCursor;
```

Comment:
I'd like there to be only one definition of current tool can you get rid of all of the other versions of it in this PR. We will need to create a way to access (get/set) the raw tool though.

Reply:
As far as I can tell, there is no difference between tool and cursor in the context of `0x00523393`. I agree it needs to be consolidated, though. Shall we move it to the `Ui` namespace and give it a getter and setter function there?

Reply:
Hmm perhaps. Its a bit annoying how most windows set it via one interface but some override that interface and access the value direct.